test/quantization/test_quantize_script.py

```diff
@@ -1668,10 +1670,15 @@ def forward(self, x):
         # and for N general value op between conv we should have
         # N + 1 quantize_per_tensor between these ops
         m1 = convert_script(m, debug=True)
-        conv_op_quant = 4
         # NB: This Needs to be updated when we add more ops to test
-        general_value_op_quant = 14
-        FileCheck().check_count("aten::quantize_per_tensor(", conv_op_quant + general_value_op_quant + 1, exactly=True) \
+        # number of quantize_per_tensor op for type
+        num_quant_by_op_type = {'conv': 2, 'common': 1, 'interpolate': 3}
```

Comment:
Can you explain why there are three quantize_per_tensor ops per interpolate op?

Reply:
there are some if statement in the code, and we have one quantize_per_tensor for each callsite of the aten ops inside if.